docs/sources/flow/reference/components/prometheus.exporter.gcp.md

```diff
@@ -0,0 +1,157 @@
+---
+# NOTE(tburgessdev, from rfratto): the title below has zero-width spaces injected into it to
```

Comment:
heads-up: this part would change a bit after this: https://github.com/grafana/agent/pull/4473